packages/react-dom/src/events/__tests__/SyntheticWheelEvent-test.js

```diff
@@ -9,61 +9,106 @@
 
 'use strict';
 
-var SyntheticWheelEvent;
-
 describe('SyntheticWheelEvent', () => {
-  var createEvent;
+  var React;
+  var ReactDOM;
+  var createMouseEvent;
 
   beforeEach(() => {
-    // TODO: can we express this test with only public API?
-    SyntheticWheelEvent = require('../SyntheticWheelEvent').default;
-
-    createEvent = function(nativeEvent) {
-      var target = require('../getEventTarget').default(nativeEvent);
-      return SyntheticWheelEvent.getPooled({}, '', nativeEvent, target);
+    React = require('react');
+    ReactDOM = require('react-dom');
+    createMouseEvent = (type = '', options = {}) => {
+      const event = document.createEvent('MouseEvent');
+      event.initEvent(type, true, false);
+      return Object.assign(event, options);
     };
   });
 
   it('should normalize properties from the Event interface', () => {
-    var target = document.createElement('div');
-    var syntheticEvent = createEvent({srcElement: target});
+    const container = document.createElement('div');
+    const event = createMouseEvent('', {srcElement: container});
+    container.dispatchEvent(event);
 
-    expect(syntheticEvent.target).toBe(target);
-    expect(syntheticEvent.type).toBe(undefined);
```

Comment:
It wouldn't be undefined in practice. The old test was odd, but we shouldn't replicate that odd behavior in the new test :-)